scripts/load_doctors_and_patients.psql

```diff
@@ -1 +1,3 @@
 -- load doctors and patients tables from csv files
+\copy doctors (family_name, given_name, gender, born_on, specialty) FROM `data/doctors.csv` WITH (FORMAT csv, HEADER true)
+\copy doctors (family_name, given_name, gender, born_on, insuarence) FROM `data/doctors.csv` WITH (FORMAT csv, HEADER true)
```

Comment:
- `\copy doctors` should be `\copy patients`.
- typo: `insuarence` should be `insurance`.
- the file paths need to be surrounded by single quotation marks instead of grave accents.